src/components/tooltip/tooltip.tsx

```diff
@@ -209,6 +231,9 @@ class Tooltip extends Component<TooltipProps & WithEventManagerProps, any> {
   componentDidMount() {
     const {eventManager} = this.props;
     eventManager!.listen(document, 'click', e => this.handleClickOutside(e));
+    if (this._buttonRef?.addEventListener) {
```

Comment:
in order to make sure it's a valid element, since for example in the big play button, the tooltip inside the button and so children is not contains the button (and this is also not relevant to this button since the tooltip is not displayed when focus on it)